nextclade/defaults/colors.tsv

```diff
@@ -1,8 +1,8 @@
 # genotypes assigned by augur clades
-clade_membership	Angola	#3F63CF
-clade_membership	East Africa	#529AB6
-clade_membership	East/Central Africa	#75B681
-clade_membership	South America I	#A6BE55
-clade_membership	South America II	#D4B13F
-clade_membership	West Africa I	#E68133
-clade_membership	West Africa II	#DC2F24
+clade_membership	Clade I	#3F63CF
+clade_membership	Clade II	#529AB6
+clade_membership	Clade III	#75B681
+clade_membership	Clade IV	#A6BE55
+clade_membership	Clade V	#DC2F24
+clade_membership	Clade VI	#E68133
+clade_membership	Clade VII	#D4B13F
```

Comment:
I don't remember if we discussed in the meeting whether we want multiple clade systems in the Nextclade dataset, i.e. should we have these numeric clades as the default, but still include the geographical genotype labels as a separate color-by?

Reply:
> I don't remember if we discussed in the meeting whether we want multiple clade systems in the Nextclade dataset, i.e. should we have these numeric clades as the default, but still include the geographical genotype labels as a separate color-by?

The impression that I took away from the feedback in that meeting was that we wanted to de-emphasize the geographically oriented names, so I would lean away from allowing color-by for them.